pkg/stanza/fileconsumer/internal/fingerprint/fingerprint.go

```diff
@@ -18,50 +19,64 @@ const MinSize = 16 // bytes
 // Fingerprint is used to identify a file
 // A file's fingerprint is the first N bytes of the file
 type Fingerprint struct {
-	FirstBytes []byte
+	firstBytes  []byte
+	HashBytes   []byte
+	BytesLength int
```

Comment:
Can you choose a better name for this? This is the amount of bytes that was used to generate the hash. I don't think `BytesLength` describes that well enough.

Reply:
How about `nBytesUsed`?